### DISCUSSION TITLE
TF footprint enrichment

### DISCUSSION THREAD

**Discussion by AnjaliC4** [stuart-lab/signac#540] [Q&A] [answered]
Hi, Could you please explain how the "expected Tn5 enrichment" is calculated or what it represents relative to Tn5 insertion enrichment in TF footprinting?
Thanks!

**Comment by timoast** [comment, marked as answer]
Hi, please see the section "Motif footprinting" in the Signac preprint: https://www.biorxiv.org/content/10.1101/2020.11.09.373613v1

**Reply by AnjaliC4** [reply]
Thanks! :)
